src/taipy/core/config/config.schema.json

```
@@ -8,7 +8,9 @@
     "CORE": {
       "description": "Data nodes",
       "type": "object",
-      "properties": {},
+      "properties": {
+        "version": "3.0.0"
```

should we have a description and a type instead of a value ?

---

do you mean something like: 
```suggestion
        {"version": 
                {"description": "The version of Taipy Core",
                "type": "string"
                }
        }
```

---

Yes, but @FredLL-Avaiga should know better.

---

yes that would do.
What about the sequences description ?